docs/src/use/configure/rules.md

```diff
@@ -142,6 +142,37 @@ You can also use this format with configuration comments, such as:
 
 ## Disabling Rules
 
+### Use with Caution
```

Comment:
I think we can use a bulleted list instead of subheadings and paragraphs for this content.